common/api-review/firestore-lite.api.md

```diff
@@ -4,6 +4,7 @@
 
 ```ts
 
+import { EmulatorMockTokenOptions } from '@firebase/util';

```

Comment:
Can you rename `Settings` to FirestoreSettings as well (for Lite)?

Reply:
Done